assembl/tests/views_tests/test_graphql.py

```diff
@@ -537,12 +538,19 @@ def test_get_question_via_node_query(graphql_request, thematic_and_question):
             __typename,
             ... on Question {
                 title
+                numPosts
+                numContributors
+                totalSentiments
             }
         }
     }""" % first_question_id, context_value=graphql_request)
     assert json.loads(json.dumps(res.data)) == {
-        u'node': {u"__typename": u"Question",
-                  u"title": u"Comment qualifiez-vous l'emergence de l'Intelligence Artificielle dans notre société ?"}}
+        u'node': {
+            u"__typename": u"Question",
+            u"numPosts": 0,
+            u"numContributors": 0,
+            u"totalSentiments": 0,
```

Comment:
shouldn't we test cases with sentiments, contributors, posts, etc ?

Reply:
I added assertions to existing test_mutation_add_sentiment tests